LibTest/isolate/Isolate/pause_A01_t01.dart

```diff
@@ -13,43 +13,56 @@
  * response to, e.g., timers or receive-port messages. When the isolate is
  * resumed, it handles the already enqueued events.
  *
- * @description Check that pause() does not stop the isolate to execute code.
```

Comment:
Actually I think I misunderstood this test and what `Isolate.pause()` was supposed to do: Pausing should take only effect when coming back to the event loop. So this test was supposed to test that pausing will actually not pause the `entryPoint` function (which is synchronous and therefore doesn't go back to event loop)

So I must apologize and the last change you made, was correct. Maybe you could revert to that revision?